### ISSUE TITLE
PVP Signet shows as Allies 

### ISSUE THREAD

**Issue by Inkraja** [Inkraja/Advanced-Combat-Tracker#4]
Signet of Protector's Wrath, shows in ACT as an Allies instead of a cast
![act_pvpsignet](https://cloud.githubusercontent.com/assets/10184036/13967486/a57dac40-f077-11e5-9cb8-1bc67f753ce6.jpg)

**Comment by JochenSchwander** [comment]
Got a log line for that signet?

**Comment by Inkraja** [comment]
right... these are some possible proc's for the signet i found in the log

```
[20:48:24] Librariam's Protector's Wrath hits (Normal) Bird of the Zero Point Pathogen for 63 physical damage. (Normal)
[20:48:25] Buff Protector's Wrath terminated on Bird of the Zero Point Pathogen.
[20:51:17] Librariam's Protector's Wrath dealt 0 damage to Bird of the Zero Point Pathogen's Psychic AEGIS.
[20:53:54] (Critical) Librariam's Protector's Wrath hits (Normal) Bird of the Zero Point Pathogen for 71 physical damage. (Normal)
[20:54:48] (Critical) Librariam's Protector's Wrath hits (Normal) Bird of the Zero Point Pathogen for 85 physical damage. (Penetrated)
[20:55:13] (Critical) Librariam's Protector's Wrath hits (Normal) Bird of the Zero Point Pathogen for 30 physical damage. (Blocked)
```

**Comment by JochenSchwander** [comment]
fixed in 1.0.6.801 pre 1.0.6.9 rev 01 dc968cf2c3f7225b840441fe603ed4a93c3a5d10
